en/docs/learn/rate-limiting/introducing-throttling-use-cases.md

```diff
@@ -56,42 +56,42 @@ Let’s look at how each of these can be important for serving requests through
+Here is a sample for configuring JWT claim condition by considering the version of the API (http://wso2.org/claims/subscribe).
 
-![](../../assets/img/learn/advanced-throttling-jwt-condition.png)
+![]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)
 ##### Query parameters
 
 Filtering based on query parameters almost always apply to HTTP GET requests when doing search type of operations. For example, if you have a search API with `category` as a query parameter, you can have different limits for searching different categories.
 
-![](../../assets/img/learn/advanced-throttling-query-conidtion.png)
+![]({{base_path}}/assets/img/learn/new-advanced-throttling-query-conidtion.png)
 Eg : 'sales' category can be allocated with more requests than 'hr' category
 
 ### Implications on applications that consume APIs
 
 #### Per token quota: Application Developer
 
-When an application developer subscribes their application to an API, they select a tier (limit) for their application to invoke the API. This limit applies across all users of the application when accessing the particular API. To ensure that a fair distribution of the quota is available among all the users, it is important to consider setting a per user quota for the application, since a user is identified by a token (in OAuth2.0, this limit is known as the per token quota). It is important to note that the limit enforced by this setting applies to a single user (token) accessing all APIs of the application. The counters maintained when evaluating a per token quota are shared across all nodes in the Gateway cluster. For information on how to define a throttling tier to an application, see [Application-level throttling (application developer)](../setting-throttling-limits/#application-level-throttling-application-developer)) .
+When an application developer subscribes their application to an API, they select a tier (limit) for their application to invoke the API. This limit applies across all users of the application when accessing the particular API. To ensure that a fair distribution of the quota is available among all the users, it is important to consider setting a per user quota for the application, since a user is identified by a token (in OAuth2.0, this limit is known as the per token quota). It is important to note that the limit enforced by this setting applies to a single user (token) accessing all APIs of the application. The counters maintained when evaluating a per token quota are shared across all nodes in the Gateway cluster. For information on how to define a throttling tier to an application, see [Application-level throttling (application developer)]({{base_path}}/learn/rate-limiting/setting-throttling-limits/#application-level-throttling-application-developer)).
 
 
 The below diagram shows how throttle policies are applied at different levels.
```

Comment:
Can we change this to "The following diagram" please

Reply:
Fixed with 8ac6250